src/main/java/org/jvnet/hudson/test/LoggerRule.java

```diff
@@ -43,25 +43,53 @@
 /**
  * A test rule which allows you to easily enable one or more loggers for the duration of a test.
  * Call {@link #record(Class, Level)} or another overload for the rule to take effect.
+ * <p>By default messages are merely printed to test output.
+ * If you also want to examine them, call TODO
```

Comment:
TODO?  (looks like you have implemented this!)
call `#capture(int)`and subsequently `#getRecords` to retreive the last n records?